wrappers/javascript/anoncreds-react-native/cpp/anoncreds.cpp

```diff
@@ -103,22 +103,242 @@ jsi::Value createCredentialDefinition(jsi::Runtime &rt, jsi::Object options) {
+
+  ErrorCode code = anoncreds_credential_offer_from_json(b, &credentialOfferP);
+  handleError(rt, code);
+
+  return jsi::Value(int(credentialOfferP));
+};
+
+jsi::Value schemaFromJson(jsi::Runtime &rt, jsi::Object options) {
+  auto json = jsiToValue<std::string>(rt, options, "json");
+
+  ObjectHandle schemaP;
+
+  ByteBuffer b = stringToByteBuffer(json);
+  
+  ErrorCode code = anoncreds_schema_from_json(b, &schemaP);
+  handleError(rt, code);
+
+  return jsi::Value(int(schemaP));
+};
+
+jsi::Value masterSecretFromJson(jsi::Runtime &rt, jsi::Object options) {
+  auto json = jsiToValue<std::string>(rt, options, "json");
+
+  ObjectHandle masterSecretP;
+
+  ByteBuffer b = stringToByteBuffer(json);
+  
+  ErrorCode code = anoncreds_master_secret_from_json(b, &masterSecretP);
+  handleError(rt, code);
+
+  return jsi::Value(int(masterSecretP));
+};
+
+jsi::Value credentialRequestFromJson(jsi::Runtime &rt, jsi::Object options) {
+  auto json = jsiToValue<std::string>(rt, options, "json");
+
+  ObjectHandle credentialRequestP;
+
+  ByteBuffer b = stringToByteBuffer(json);
+
+  ErrorCode code = anoncreds_credential_request_from_json(b, &credentialRequestP);
+  handleError(rt, code);
+
+  return jsi::Value(int(credentialRequestP));
+};
+
+jsi::Value credentialRequestMetadataFromJson(jsi::Runtime &rt, jsi::Object options) {
+  auto json = jsiToValue<std::string>(rt, options, "json");
+
+  ObjectHandle credentialRequestMetadataP;
+
+  ByteBuffer b = stringToByteBuffer(json);
+
+  ErrorCode code = anoncreds_credential_request_metadata_from_json(b, &credentialRequestMetadataP);
+  handleError(rt, code);
+
+  return jsi::Value(int(credentialRequestMetadataP));
+};
+
+jsi::Value credentialFromJson(jsi::Runtime &rt, jsi::Object options) {
+  auto json = jsiToValue<std::string>(rt, options, "json");
+
+  ObjectHandle credentialP;
+
+  ByteBuffer b = stringToByteBuffer(json);
+
+  ErrorCode code = anoncreds_credential_from_json(b, &credentialP);
+  handleError(rt, code);
+
+  return jsi::Value(int(credentialP));
+};
+
+jsi::Value revocationRegistryDefinitionPrivateFromJson(jsi::Runtime &rt, jsi::Object options) {
+  auto json = jsiToValue<std::string>(rt, options, "json");
+
+  ObjectHandle revocationRegistryDefinitionPrivateP;
+
+  ByteBuffer b = stringToByteBuffer(json);
+
+  ErrorCode code = anoncreds_revocation_registry_definition_private_from_json(b, &revocationRegistryDefinitionPrivateP);
+  handleError(rt, code);
+
+  return jsi::Value(int(revocationRegistryDefinitionPrivateP));
+};
+
+jsi::Value revocationStateFromJson(jsi::Runtime &rt, jsi::Object options) {
+  auto json = jsiToValue<std::string>(rt, options, "json");
+
+  ObjectHandle revocationStateP;
+
+  ByteBuffer b = stringToByteBuffer(json);
+
+  ErrorCode code = anoncreds_revocation_state_from_json(b, &revocationStateP);
+  handleError(rt, code);
+
+  return jsi::Value(int(revocationStateP));
+};
+
+jsi::Value revocationRegistryDeltaFromJson(jsi::Runtime &rt, jsi::Object options) {
```

Comment:
We don't have deltas anymore in the public API right?

Reply:
Right. This was based on FFI interface in NodeJs, where it was still present. So I think we'll need to remove it from there as well.

Reply:
I see this method is still exposed in FFI, so I think we can keep it even if not used in regular API flow.